### ISSUE TITLE
Problem uploading some particular videos/pics

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#574]
@Woseseltops , see earlier emails from Julie. Now looking at https://aslsignbank.haskins.yale.edu/dictionary/gloss/2877/, the video doesn't display. Julie didn't compress this video any differently than other ones. Can you check what might be wrong?
Another example was https://aslsignbank.haskins.yale.edu/dictionary/gloss/2893/, but i understand there are more. As ASL SB indicates that they are uploaded but that it simply cannot display them, perhaps you can find more once the problem is clear.

**Comment by ocrasborn** [comment]
The gist of my last email response to julie from Aug 2018: when I uploaded the these videos in Global SB, everything went well.

**Comment by Woseseltops** [comment]
Ah yes, I looked at this problem before and couldn't solve it then. I'll give it another try right now.

**Comment by Woseseltops** [comment]
Okay, I already figured it out! All videos are organized in subfolders based on their first two characters, so the videos for the two glosses above, EUROPE and ZILCH, are in EU and ZI respectively. I noticed that unlike most other videos, EU and ZI had no execute permissions for the 'other' users (like the webserver user), along with the folders AH, CZ, EC, MC, OI, PS, QT and ZA. I made permissions consistent, and now the videos work. 

@vanlummelhuizen  mentioned that this is related to issue
